### PR TITLE
Cleanup Overview and Running Gradle section based on Github issues

### PR DESCRIPTION
**Pull request by lkasso**
### Context
Handles Github issues and a few internal issues in sections that have been recently updated in the Gradle User Manual.

This is a documentation change ONLY.

### Fixes:
- Gradle no demon flag - https://github.com/gradle/gradle/issues/25272 
- Gradle wrapper update - https://github.com/gradle/gradle/issues/12273
- Gradle demon desc - https://github.com/gradle/gradle/issues/21676 
- Gradle properties - https://github.com/gradle/gradle/issues/24314
- New admonition icons - https://github.com/gradle/gradle/issues/22978 
- Pushed https://github.com/gradle/gradle/pull/25008 from https://github.com/gradle/gradle/issues/24508 which was never merged
- Formatting issue in composite_builds.adoc found by @abstratt 
- Remove double footnote in tutorial_using_tasks.adoc found by @alllex 
- Fix custom CSS for collapsible blocks found by @alllex 
- Fix issue in tutorial part 3 found by community member and submitted through feedback form

### Reviewers:
Ready for Merge. Build Distribution relevant links:
- [Configuring the Build Environment](https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/70436400:id/distributions/gradle-8.4-docs.zip!/gradle-8.4-20230817175936%2B0000/docs/userguide/build_environment.html)
- [Gradle Wrapper](https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/70436400:id/distributions/gradle-8.4-docs.zip!/gradle-8.4-20230817175936%2B0000/docs/userguide/gradle_wrapper.html)
- [Gradle Daemon](https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/70436400:id/distributions/gradle-8.4-docs.zip!/gradle-8.4-20230817175936%2B0000/docs/userguide/gradle_daemon.html)
- New Admonition icon examples:
  - [NOTE, WARNING and IMPORTANT](https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/70436400:id/distributions/gradle-8.4-docs.zip!/gradle-8.4-20230817175936%2B0000/docs/userguide/performance.html)
  - [TIP and IMPORTANT](https://builds.gradle.org/repository/download/Gradle_Master_Check_BuildDistributions/70436400:id/distributions/gradle-8.4-docs.zip!/gradle-8.4-20230817175936%2B0000/docs/userguide/configuration_cache.html)

### CODE REVIEW

**Comment by lkasso** [comment]
@bot-gradle test BD

**Comment by lkasso** [comment]
@bot-gradle test BD

**Comment by lkasso** [comment]
@bot-gradle test BD

File: subprojects/docs/src/docs/userguide/authoring-builds/tutorial_using_tasks.adoc [hunk 1]
```diff
@@ -44,13 +45,13 @@ Still, to give you an idea of what a task is, we will look at defining some simp
 [[sec:hello_world]]
 == Hello world
 [.multi-language-text.lang-kotlin]
-You run a Gradle build using the `gradle` command. The `gradle` command looks for a file called `build.gradle.kts` in the current directory.footnote:[There are command line switches to change this behavior. See <<command_line_interface.adoc#command_line_interface,Command-Line Interface>>] We call this `build.gradle.kts` file a _build script_, although strictly speaking it is a build configuration script, as we will see later. The build script defines a project and its tasks.
+You run a Gradle build using the `gradle` command. The `gradle` command looks for a file called `build.gradle.kts` in the current directory.{fn-disclaimer} We call this `build.gradle.kts` file a _build script_, although strictly speaking it is a build configuration script, as we will see later. The build script defines a project and its tasks.
```
[the remark this answers is not in the document]
Comment (lkasso) [reply subprojects/docs/src/docs/userguide/authoring-builds/tutorial_using_tasks.adoc:48]: This is a file currently under an update in a separate PR. I will make sure to include your requested changes in the separate PR.

**Review by akshayabd — APPROVED** [review]

**Comment by lkasso** [comment]
@bot-gradle test and merge

**Pull request merged**